app/views/govuk_publishing_components/components/_feedback.html.erb

```diff
@@ -81,8 +92,19 @@
     </div>
   </form>
 
-  <form action="/contact/govuk/email-survey-signup" id="page-is-not-useful" class="gem-c-feedback__form gem-c-feedback__form--email gem-c-feedback__js-show js-feedback-form js-hidden" data-track-category="user_satisfaction_survey" data-track-action="banner_taken" method="post">
-    <a href="#" class="gem-c-feedback__close js-close-form" aria-controls="page-is-not-useful" aria-expanded="true" role="button">Close</a>
+  <form action="/contact/govuk/email-survey-signup"
+    id="page-is-not-useful"
+    class="gem-c-feedback__form gem-c-feedback__form--email gem-c-feedback__js-show js-feedback-form js-hidden"
+    data-track-category="user_satisfaction_survey"
+    data-track-action="banner_taken"
+    method="post">
+    <a href="#"
+      class="gem-c-feedback__close js-close-form"
+      data-track-category="yesNoFeedbackForm"
```

Comment:
This doesn't seem to be specified in the Trello ticket, might be worth asking Tim if this is correct.

Would it make sense that this would be something like category `user_satisfaction_survey`, action `form_closed`?

In any case this would be the only remaining action for the `yesNoFeedbackForm` category - seems wrong?

Reply:
See the comments in the Trello ticket, have discussed with Tim.